__tests__/lines.js

```diff
@@ -0,0 +1,15 @@
+module.exports = [
```

Comment:
This shouldn’t be necessary — your plugin shouldn’t need to split the text into lines to process it, and you should try to stick to just reading from the AST whenever possible.

Reply:
I'll remove this as per my prev comment